SQLScripts/InsertData_ATeam.sql

```diff
@@ -1,14 +1,167 @@
-VALUES ("Product Principal", "PlaceHolder", "2021-07-13", "Full Time", "Birmingham", 1, 1, ""),
-		("Solutions Architect",
-		 "As a Solution Architect (Manager) in Kainos, you’ll be responsible for leading multi-skilled agile teams to design and deliver high quality solutions which delight our customers and impact the lives of users worldwide. You’ll work with customer architects to agree functional and non-functional designs, advising customers and managers on the estimated effort, technical implications and complexity surrounding your designs.",
-		  "2021-06-20", "Full Time", "Derry-Londonderry", 1, 1, "https://kainossoftwareltd.sharepoint.com/people/Job%20Specifications/Forms/AllItems.aspx?id=%2Fpeople%2FJob%20Specifications%2FEngineering%2FJob%20Profile%20%2D%20Solution%20Architect%20%28Manager%29%2Epdf&parent=%2Fpeople%2FJob%20Specifications%2FEngineering&p=true&originalPath=aHR0cHM6Ly9rYWlub3Nzb2Z0d2FyZWx0ZC5zaGFyZXBvaW50LmNvbS86YjovZy9wZW9wbGUvRVNwcDU2Zlp5cmhLaTZ1Q3V3cHNoRHdCM25tbzFCTWtiUUtZRFhiTkhDaV9zdz9ydGltZT1tcHlFbzJSczJVZw"),
-        ("Market Intelligence Analyst", "PlaceHolder", "2021-08-24", "Full Time", "London", 1, 1, ""), 
-        ("Market Intelligence Analyst", "PlaceHolder", "2021-01-24", "Full Time", "Birmingham", 2, 2, "");
\ No newline at end of file
+VALUES ("Product Principal", "PlaceHolder", "13-07-2021", "Full Time", "Birmingham", 1, 1, ""),
+        ("Solutions Architect",
+         "As a Solution Architect (Manager) in Kainos, you’ll be responsible for leading multi-skilled agile teams to design and deliver high quality solutions which delight our customers and impact the lives of users worldwide. You’ll work with customer architects to agree functional and non-functional designs, advising customers and managers on the estimated effort, technical implications and complexity surrounding your designs.",
+          "20-06-2021", "Full Time", "Derry-Londonderry", 1, 1, "https://kainossoftwareltd.sharepoint.com/people/Job%20Specifications/Forms/AllItems.aspx?id=%2Fpeople%2FJob%20Specifications%2FEngineering%2FJob%20Profile%20%2D%20Solution%20Architect%20%28Manager%29%2Epdf&parent=%2Fpeople%2FJob%20Specifications%2FEngineering&p=true&originalPath=aHR0cHM6Ly9rYWlub3Nzb2Z0d2FyZWx0ZC5zaGFyZXBvaW50LmNvbS86YjovZy9wZW9wbGUvRVNwcDU2Zlp5cmhLaTZ1Q3V3cHNoRHdCM25tbzFCTWtiUUtZRFhiTkhDaV9zdz9ydGltZT1tcHlFbzJSczJVZw"),
+        ("Market Intelligence Analyst", "PlaceHolder", "24-08-2021", "Full Time", "London", 1, 1, ""),
+        ("Market Intelligence Analyst", "PlaceHolder", "24-01-2021", "Full Time", "Birmingham", 2, 2, "");
+
+INSERT INTO competency(competencyName, competencyDescription)
+VALUES ("Personal Performance", "Effective leadership requires individuals to draw upon values, strengths, and abilities to deliver high standards of service. Demonstratingeffectiveself-awareness,self-management, continuedpersonal development and actingwith integrity.");
```

Comment:
Copy/paste has let you down here! Some missing spaces between words.

Since the job is to facilitate entry and presentation of the data, rather than to enter the data, this isn't a big issue – but when you present the feature to the customer it could look bad if there are little mistakes like that. It could give an impression of carelessness.